deepmd/entrypoints/test.py

```diff
@@ -417,8 +419,20 @@ def test_ener(
     mae_e = mae(diff_e)
     rmse_e = rmse(diff_e)
     diff_f = force - test_data["force"][:numb_test]
-    mae_f = mae(diff_f)
-    rmse_f = rmse(diff_f)
+    if find_atom_pref == 1:
+        atom_pref = test_data["atom_pref"][:numb_test]
```

Comment:
```suggestion
        atom_pref = test_data["atom_pref"][:numb_test].astype(bool)
```
@njzjz Is this what you mean?

Reply:
No, I mean `atom_pref` was not designed (when it was proposed in #41) as the mask. One may use it for other purposes and doesn't expect to see a masked result.

Reply:
And in that case, we can simply convert them to bool type.